templates/jinja2/index.html

```diff
@@ -20,17 +20,7 @@
         <strong>Dear BGA readers,</strong>
```

Comment:
if we can use html throughout, is there a reason to not include this in the message?

Reply:
why would it prevent any extra styling? we can't have a "class" be set in the user input html?

Reply:
@fgregg we can very easily, i'm just not sure the technical level of whoever will be inputting this code? do they know bootstrap?

Reply:
they probably don't know bootstrap, but we can tell them that the "BGA reader" section should have a class in it, and they can follow that.

Reply:
@fgregg removed the greeting with this commit: c982b43. would you want to do that with the donate button, too?